src/components/menu/styled-components.js

```diff
@@ -11,34 +11,88 @@ type StyledListItemPropsT = {
   $isHighlighted: boolean,
 } & StyledPropsT;
 
-export const List = styled('ul', ({$theme}: StyledPropsT) => ({
+const getSharedListStyle = ($theme: ThemeT) => ({
   position: 'relative',
-  display: 'flex',
-  flexDirection: 'column',
   margin: 0,
   background: $theme.colors.white,
   borderRadius: $theme.borders.radius300,
   boxShadow: $theme.lighting.shadow600,
-  paddingTop: $theme.sizing.scale200,
-  paddingBottom: $theme.sizing.scale200,
-  paddingLeft: $theme.sizing.scale400,
-  paddingRight: $theme.sizing.scale400,
+});
+
+export const List = styled('ul', ({$theme}: StyledPropsT) => ({
+  ...getSharedListStyle($theme),
+  padding: $theme.sizing.scale600,
 }));
 
 export const ListItem = styled(
   'li',
   ({$theme, $isHighlighted}: StyledListItemPropsT) => ({
-    ...$theme.typography.font400,
+    ...$theme.typography.font300,
     position: 'relative',
     display: 'block',
     color: $isHighlighted ? $theme.colors.primary : $theme.colors.black,
-    margin: 0,
     cursor: 'pointer',
     transitionProperty: 'color',
     transitionDuration: $theme.animation.timing100,
     transitionTimingFunction: $theme.animation.easeOutCurve,
     ':hover': {
       color: $theme.colors.primary400,
     },
+    ':not(:last-child)': {
+      marginBottom: $theme.sizing.scale600,
+    },
   }),
 );
+
+export const ListProfile = styled('ul', ({$theme}: StyledPropsT) => ({
+  ...getSharedListStyle($theme),
+  padding: $theme.sizing.scale800,
+}));
+
+export const ListItemProfile = styled('li', {
+  position: 'relative',
+  display: 'flex',
+  alignItems: 'center',
+  ':not(:last-child)': {
+    marginBottom: '18px',
+  },
+});
```

Comment:
Probably already on your to-do list, but it seems like we're missing hover styles and cursor: pointer for these options.

Reply:
Yeah I need to sync with @nguyensomniac on that.